* Add a space after -o because ld on Darwin wants the argument to be
  separated from the option.

--- cccc/posixgcc.mak
+++ cccc/posixgcc.mak
@@ -29,7 +29,7 @@
 CFLAGS=-c -I../pccts/h $(CFLAGS_DEBUG) -x c++ 
 C_OFLAG=-o
 LDFLAGS=$(LDFLAGS_DEBUG)
-LD_OFLAG=-o
+LD_OFLAG=-o 
 OBJEXT=o
 CCCC_EXE=cccc
 
